frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -65,7 +65,8 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample, builderImages }) =>
         const buildTools: BuildType[] = await gitService.detectBuildTypes();
         setFieldValue('image.isRecommending', false);
         const buildTool = buildTools.find(
-          ({ buildType: recommended }) => recommended && builderImages.hasOwnProperty(recommended),
+          ({ buildType: recommended }) =>
+            builderImages && recommended && builderImages.hasOwnProperty(recommended),
```

Comment:
If there's no `builderImages` why even run the detection?

Reply:
Done